### DISCUSSION TITLE
Why does Wandb revert to a single dot at the end?

### DISCUSSION THREAD

**Discussion by cinjon** [axolotl-ai-cloud/axolotl#1568] [Q&A]
When my axolotl model finishes finetuning, it becomes a single dot on wandb. I'm not doing anything special. My config looks like this, where the nulls are detritus from auto-saving configs:

```
wandb_entity: <entityname>
wandb_log_model: null
wandb_name: <unique_wandbname>
wandb_project: <projectname>
wandb_run_id: null
wandb_watch: null
```

In the wandb log file, it shows all the step-logs as expected.

![Screenshot 2024-04-25 at 12 07 06 PM](https://github.com/OpenAccess-AI-Collective/axolotl/assets/615351/abdee80e-92f3-409d-a408-f41e44c99b4d)

**Comment by NanoCode012** [comment]
It's not a revert. The loss will appear on the second page :) . This one dot is for a different metric.

These days, this will appear properly now.
